server/src/user/user.service.ts

```diff
@@ -21,4 +22,17 @@ export class UserService {
   async getUsers(): Promise<User[]> {
     return await this.userRepository.createQueryBuilder().limit(50).getMany();
   }
+
+  async updateUser(userId: string, body: UserDto): Promise<void> {
+    try {
```

Comment:
Shouldn't use try..catch. In case of error, the user will not see any response and the connection will timeout!